### DISCUSSION TITLE
Centralised way to hotreload our pages created using Markup

### DISCUSSION THREAD

**Discussion by SarthakB26** [CommunityToolkit/Maui.Markup#225] [General]
What would be a decent way to add HotReload functionality to our pages created with Markup

I know there is a way to subscribe when HotReload is hit manually then we need to re execute our whole content creation method and set it to Content of page right but can we do this to a base page or something

We should add this to sample app as well I guess

**Comment by pictos** [comment]
@drasticactions, since you work on VS tooling do you know what we can improve on this lib in order to make it better for C# hotreload?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
